bundles/azure/aso/psql/bundle/config/01-flexible-server.ytt.yml

```diff
@@ -0,0 +1,248 @@
+#@ load("@ytt:data", "data")
+#@ load("@ytt:assert", "assert")
+#@ load("@ytt:regexp", "regexp")
+#@ load("@ytt:ip", "ip")
+
+
+#@ flexibleServerName    = data.values.name
+#@ dbSecretName          = data.values.name
+#@ resourceGroupK8SName  = data.values.name
+#@ serviceAccountName      = data.values.name + "-reader"
+#@ serviceAccountNamespace = data.values.namespace
+#@ roleName                = data.values.name + "-reading"
+
+#@ mandatory_fields = [ "name", "namespace", "location", "firewall_rules", "aso_controller_namespace" ]
```

Comment:
_nit_: I'd still prefer `namespace` not to be mandatory

Reply:
It is mandatory for ytt to have a value otherwise it will break. We're gonna set a default value for "namespace" (as well as for other fields) which will make it no longer mandatory for the user, but for the sake of consistency I'd rather keep it in this check.

Reply:
See: https://github.com/vmware-tanzu/tanzu-application-platform-reference-service-packages/pull/22#discussion_r976241699

But it's a _nit_, thus not blocking.

Reply:
Done

Reply:
Sorry for the misunderstanding, see replies [here](https://github.com/vmware-tanzu/tanzu-application-platform-reference-service-packages/pull/22#discussion_r977873259)